cfme/tests/services/test_provision_stack.py

```diff
@@ -200,7 +200,6 @@ def test_remove_template_provisioning(provider, provisioning, catalog, catalog_i
     template.delete()
     row_description = 'Provisioning Service [{}] from [{}]'.format(item_name, item_name)
     cells = {'Description': row_description}
-    wait_for(lambda: requests.find_request(cells), num_sec=500, delay=20)
```

Comment:
added if_exists for such cases. but this particular doesn't need it at all - wait_for_request does all the same. so saving time a bit